module/data/actor/templates/sentient.mjs

```diff
@@ -37,6 +37,22 @@ export default class SentientTemplate extends CommonTemplate {
                 initialKeysOnly: true,
                 label: "ED.Attributes.attributes"
             } ),
+            healthRate: new fields.SchemaField( {
```

Comment:
What is that for? The token bar values? If so, this should probably go into a `tokenData` `SchemaField` or something like that. So it's clear what it stands for.
Also, why is that necessary for tokens? We have the health ratings elsewhere don't we?